src/app/containers/Timestamp/index.jsx

```diff
@@ -29,22 +30,59 @@ const formatTimestamp = dateObj => {
   return `${dayNumeric} ${monthLong} ${fullYear}`;
 };
 
-const TimestampContainer = ({ timestamp }) => {
-  const dateObj = new Date(timestamp);
+const isTenHoursAgo = milliseconds => {
+  const now = Date.now();
+  return now - milliseconds >= 10 * 60 * 60 * 1000;
+};
+
+const updatedTimestamp = (dateObj, milliseconds) => (
+  <Timestamp datetime={formatDateTime(dateObj)} prefix="Updated">
+    {milliseconds}
+  </Timestamp>
+);
+
+const hasBeenUpdated = (updated, published) => updated !== published;
+
+const createSecondTimestamp = (updated, dateObj) =>
```

Comment:
This is quite tricky to read IMO, not overly clear whats going on, could maybe make this a bit clear and combine with lines 67-71 to be self contained.

Reply:
Also not sure why you pass (updated, dateObj) when dateObj contains updated (or atleast a date type of it)

Reply:
Maybe something like
```
const updatedTimestamp = updated => {
  if (!updated) {
    return null;
  }

  if (isTenHoursAgo(updated)) {
    updatedTimestamp(updated, formatTimestamp(updated));
  }

  updatedTimestamp(updated, relativeTime(updated));
};
```

And then

```
return (
    <Fragment>
      <Timestamp datetime={formatDateTime(publishDate)}>
        {formatTimestamp(publishDate)}
      </Timestamp>
      {updatedTimestamp()}
    </Fragment>
  );
```

Reply:
> Also not sure why you pass (updated, dateObj) when dateObj contains updated (or atleast a date type of it)

I did that because I wanted the distinction between the timestamp and the Date object to be clear, without repeating `new Date();` everywhere

Reply:
Im more of a fan of passing around the unix timestamp and then converting it into a Date where used tbh